### DISCUSSION TITLE
[Q&A] 調整左上角 "LobeHub" Logo 圖示的方法與圖檔位置資訊請求。

### DISCUSSION THREAD

**Discussion by Sen-CaPoo** [lobehub/lobehub#853] [Development Question] [answered]
### 🧐 问题描述 | Proposed Solution

根據 UI 風格，左上方會有 "LobeHub" 的白底黑字與黑底白字的 Logo 圖示，我有自行調整 UI 的整體顏色，需要一起調整這個 Logo 的大小和顏色，但找不到相關的修改位置與方法，請求協助，感謝。

### 📝 补充信息 | Additional Information

可能的參考檔案，但由於這是屬於一個套件，我修改後好像無法真正的套用，這部分我比較不熟悉，這邊資訊僅提供參考。
File: @lobehub\ui\es\Logo\LogoText.js

_No response_

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
这个 Logo 在 lobe-ui 组件库中：https://github.com/lobehub/lobe-ui/blob/master/src/Logo/index.tsx

[a comment by an automated account is not shown]
